### PR TITLE
[Enhancement] Move GCP integration from experimental to GA

### PR DESCRIPTION
**Pull request by suyograo**
## What does this PR do?

Move GCP integration to GA from experimental. This will be used in the upcoming Dataflow integration project.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Comment by suyograo** [comment]
Not sure whats up with the CLA  comment :)

File: packages/gcp/manifest.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 name: gcp
 title: Google Cloud Platform (GCP)
-version: 0.3.3
-release: experimental
```
Comment (andrewkroh) [inline packages/gcp/manifest.yml:4]: I was trying to figure out what the default value is for `release`. But found that the package-spec indicates that `release` is a required. So I think we want `release: ga` here.

https://github.com/elastic/package-spec/blob/a7394d0c12e4a90c3d3a4325b5a0ded27ac3fb34/versions/1/manifest.spec.yml#L282

Comment (andrewkroh) [reply packages/gcp/manifest.yml:4]: That's also what CI is complaining about.

Comment (suyograo) [reply packages/gcp/manifest.yml:4]: thanks, added

**Comment by suyograo** [comment]
@andrewkroh can you take another look?

**Comment by andrewkroh** [comment]
@suyograo I think the CLA is complaining because your commits don't contain an email. https://github.com/elastic/integrations/pull/1568.patch

> From: Suyog Rao <>

**Review by andrewkroh — APPROVED** [review]

**Comment by suyograo** [comment]
@andrewkroh thanks didn't realize the email issue. I amended the commits and looks good. thats what happens when you don't code for a while ;)

**Comment by sayden** [comment]
@endorama FYI

**Pull request merged**